submissions/chabuChabu/html-forms/product.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
```

Comment:
Remove the images in the htmlForms folder. images should be in the introToHTML folder.